guides/common/modules/proc_distributing-keys-for-remote-execution.adoc

```diff
@@ -1,7 +1,9 @@
 [id="Distributing_SSH_Keys_for_Remote_Execution_{context}"]
 = Distributing SSH Keys for Remote Execution
 
-To use SSH keys for authenticating remote execution connections, you must distribute the public SSH key from {SmartProxy} to its attached hosts that you want to manage.
+For {SmartProxies} in `ssh` mode, remote execution connections are authenticated using SSH.
+In most cases, the public SSH key from {SmartProxy} must be distributed to its attached hosts that you want to manage.
```

Comment:
What do you mean by "most cases"?

Reply:
Below there's a note "If the hosts are running on Amazon Web Services, enable password authentication." If you're using password authentication,  I think that would be the exception to "most cases."  (check me @adamruzicka )

Reply:
This feels a bit like playing with words. To be pedantic, you don't *have to* distribute the keys, but if you want to use REX, you have to make it work somehow. Using keys just happens to be the preferred way.